website/docs/pact_nirvana/step_2.md

```diff
@@ -1,15 +1,28 @@
 ---
-title: 2. Spike
+title: 2. Talk - get team alignment
+sidebar_label: 2. Talk - get team alignment
 ---
 
-If you have not used Pact before, consider trying it out on a [spike](https://en.wikipedia.org/wiki/Spike_%28software_development%29) application first.
+Contract tests are a big win when it comes to enabling teams to work and deploy independently, but they also require some level of coordination. Consumer contract tests are of very little value unless they're verified against the provider, and a provider can't write contract tests for their system without working with the consumer team to get the tests added to the consumer project.
 
-1. Using the technology stacks of your consumer and provider
+Contracts are not a replacement for good communication between or within teams. In fact, contracts require collaboration and communication. One could make the argument that this is one of the main reasons to leverage Pact and enforce communication pathways in large internal and external development organizations.
```

Comment:
Question: what does "internal and external development organizations" mean in this case? When I first read, I thought it was referring to perhaps a contract that is established between company A and company B via a service/API they share (one being consumer and the other provider). Is that right?

By the way, I understand this was just moved over, so feel free to skip this question 😬